examples/glasplaat.json

```diff
@@ -0,0 +1,171 @@
+{
+  "@context": [
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "id": "https://raw.githubusercontent.com/viaacode/iiif-manifests/IIIF-52-simple-example/examples/glasplaat.json",
+  "type": "Manifest",
+  "behavior": [
+    "individuals"
+  ],
+  "label": {
+    "nl": [
+      "Vlakbeelden Midden Rijk"
+    ]
+  },
+  "metadata": [
+    {
+      "label": {
+        "en": [
+          "Type"
+        ]
+      },
+      "value": {
+        "en": [
+          "Photo"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Date"
+        ]
+      },
+      "value": {
+        "none": [
+          "1900/1960 <dcterms:created^edtf>"
```

Comment:
ik denk dat die `<dcterms:created^edtf>` hier niet moet staan? het is niet human-readable.